src/components/Reactions/Reactions.tsx

```diff
@@ -14,16 +15,48 @@ type FilledComment = Comment & {
   };
 };
 
+type FilledLike = Like & {
+  author: {
+    email: string;
+  };
+};
+
 type Props = {
   comments: FilledComment[];
+  likes: FilledLike[];
+  onSubmitNewLike: Function;
 };
 
-const Reactions: React.FC<Props> = ({comments}: Props) => {
+const Reactions: React.FC<Props> = (props: Props) => {
+  const { loggedUser } = useContext(AuthContext);
+  const [ alreadyLiked, setAlreadyLiked ] = useState(undefined);
+
+  const likes = props.likes ? props.likes.length : 0;
+
+  useEffect(() => {
+    if(loggedUser) {
+      setAlreadyLiked(
+        props.likes.find(like => like.author.email === loggedUser?.email)
```

Comment:
you want a boolean here right?
use some
props.likes.some(like => like.author.email === loggedUser?.email)

Reply:
On really... not. If I catch just the boolean of this, I can't send the Like id... But I can make another setState, to receive the like details, and this setState, receive just the boolean...